### ISSUE TITLE
Crash when `(below|above)WidgetLimit` is 0 

### ISSUE THREAD

**Issue by ItzSkyReed** [42atomys/fabric-advancementinfo-reloaded#10]
Crash report: https://mclo.gs/zGAYSDs
config: 
```json5
{
	"displaySidebar": true,
	"displayDescription": true,
	"alphabeticOrder": true,
	"backgroundStyle": "ACHIEVEMENT",
	"headerHeight": 32,
	"footerHeight": 32,
	"criteriasWidth": 150,
	"aboveWidgetLimit": 0,
	"belowWidgetLimit": 0
}
```
it is possible to set 0 while editing config inside minecraft

**Comment by ItzSkyReed** [comment]
https://github.com/42atomys/fabric-advancementinfo-reloaded/blob/171a71250b56f47e5837301ed9f1e55019cd179b/src/main/java/codes/atomys/advancementinforeloaded/AdvancementInfoReloadedConfigModel.java#L31-L34

as far as I understand, it needs to be min = 1

**Comment by 42atomys** [comment]
Hi thanks for your multiple contributions, 
I will check, the reason to allow set as zero is to disable the above or bellow display. 

Stay tuned ✨

**Comment by 42atomys** [comment]
Hi again, the issue are resolved in **v0.4.1**, available on modrinth right now https://modrinth.com/mod/advancements-reloaded
